### PR TITLE
2016-08-02-Bitfinex Article

### PR DESCRIPTION
**Pull request by EvgDubrovin**

### CODE REVIEW

**Comment by EvgDubrovin** [comment]
> `The trading price of Bitcoin declined by 20% after the Bitfinex hack.` ❌

added a source link (https://www.coindesk.com/markets/2016/08/02/bitcoin-drops-nearly-20-as-exchange-hack-amplifies-price-decline/)

**Comment by EvgDubrovin** [comment]
> `Bitfinex implemented BitGo's multiple-signature security system to safeguard the funds.` ❌

added a source link (https://en.wikipedia.org/wiki/2016_Bitfinex_hack)

**Comment by EvgDubrovin** [comment]
> `The identity of the thief still remains unknown.` ❌

according to the source I attached, it's true (https://www.wired.com/story/security-lapses-at-hacked-crypto-exchange-bitfinex/#:~:text=But%20the%20perpetrator%20of%20the%20hack%20is%20still%20at%20large.%C2%A0). Lichtenstein and his wife are only launderers, they aren't charged in the hack itself.

**Comment by EvgDubrovin** [comment]
> `The stolen funds were consolidated into one wallet address: bc1qazcm763858nkj2dj986etajv6wquslv8uxwczt.` ❌

It's written in a plenty of articles. Here is the link to one of them (https://blog.merklescience.com/hacktrack/hack-track-bitfinex-hack-2016-recent-fund-movement-analysis#:~:text=more%20than%2094,address%20is%20bc1qazcm763858nkj2dj986etajv6wquslv8uxwczt.)

**Comment by EvgDubrovin** [comment]
> `Bitfinex resumed trading and restored service on August 10, 2016, with additional security measures.` ❌

Bitfinex wrote about it in their announcement, the link to which I attached (https://blog.bitfinex.com/announcements/site-update/#:~:text=to%20restore%20service%20after%20the%20incident%20on%20August%202nd.%20Exchange%20trading%20will%20be%20enabled%20for%20all%20currencies%20and%20pairs%2C%20while%20deposits%20and%20withdrawals%20will%20be%20enabled%20for%20BTC%2C%20ETC%2C%20ETH%2C%20and%20USD%20%E2%80%93%20with%20LTC%20and%20Tether%20to%20follow%20shortly%20thereafter.)

**Comment by EvgDubrovin** [comment]
> `Bitfinex announced full recovery from the hack on April 3, 2017, by redeeming all issued and outstanding BFX tokens.` ❌

It's written in the link I attached: (https://www.bitfinex.com/posts/198#:~:text=redeeming%20100%25%20of%20all%20currently%20issued%20and%20outstanding%20BFX%20tokens.%20This%20will%20be%20the%20final%20redemption%20of%20BFX%20tokens) and (https://www.bitfinex.com/posts/198#:~:text=full%20recovery%20in%20just%20eight%20months.)

**Comment by EvgDubrovin** [comment]
> `Bitfinex received stolen BTC from the U.S. government on February 25, 2019.` ❌

It's written in the article I attached (https://medium.com/bitfinex/bitcoins-returned-to-bitfinex-by-u-s-government-51fe84e8bb12#:~:text=Bitfinex%20are%20pleased%20to%20announce%20that%2027.66270285%20BTC%20that%20were%20unlawfully%20taken%20in%20the%20August%202016%20security%20breach%20have%20been%20received%20by%20Bitfinex%20from%20the%20U.S.%20government%20as%20the%20result%20of%20U.S.%20law%20enforcement%20efforts.) and in many other sources.

**Comment by EvgDubrovin** [comment]
> `Bitfinex offered a $400 million reward on August 4, 2020, for information leading to the identification of hackers involved in the attack.` ❌

It's written in the source I attached: (https://www.bitfinex.com/posts/494#:~:text=Bitfinex%20is%20offering%20a%20reward%20to%20any%20persons%20that%20connect%20us%20with%20hackers%20responsible%20for%20the%20unauthorized%20transfer) and (https://www.bitfinex.com/posts/494#:~:text=The%20aggregate%20rewards%20available%20under%20this%20programme%20could%20be%20worth%20up%20to%20approximately%20US%24400%20million%20at%20the%20current%20BTC%20price). It's the Bitfinex's statement.

**Comment by EvgDubrovin** [comment]
> `Bitfinex recovered stolen BTC in collaboration with Poloniex on December 14, 2021.` ❌

It's written in the source I attached (https://www.bitfinex.com/posts/746#:~:text=The%20recovery%20of%20the%20stolen%20BTC%20was%20done%20in%20collaboration%20with%20Poloniex). It's the Bitfinex's statement.

**Comment by EvgDubrovin** [comment]
> `94,643.29 BTC, more than 79% of the stolen funds, were transferred and consolidated into a single wallet on February 1, 2022.` ❌

It's written in the source I attached (https://news.bitcoin.com/3-6-billion-in-bitcoin-from-the-2016-bitfinex-hack-consolidates-into-a-single-address/#:~:text=More%20Than%2079%25%20of%20the%202016%20Stolen%20Bitfinex%20Bitcoin%20Moves%20Into%201%20Address). It's the bitcoin.com article with a variety of other sources.

**Comment by EvgDubrovin** [comment]
> `The U.S. Department of Justice recovered approximately 94,636 stolen BTC on February 7, 2022.` ❌

It's written in the source I attached - the  U.S. Department of Justice statement of facts. See point 52. Also, there are other sources confirming this info.

**Comment by JediFaust** [comment]
image

File: content/attacks/posts/2016-08-02-Bitfinex.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- **February 7, 2022:** U.S. Department of Justice [recovered](https://www.justice.gov/opa/press-release/file/1470186/download) approximately 94,636 stolen BTC, worth US$3.629 billion at the moment.
+
+## Security Failure Causes
+
+**Violation of the Security Protocol:** According to OCCRP report, Bitfinex [failed](https://www.occrp.org/en/blog/17670-confidential-report-flags-bitfinex-security-lapses-in-huge-2016-hack) to implement operational, financial, and technological controls proposed by its digital security partner Bitgo. Critical errors were made, such as storing two out of three security keys on the same device, providing full access to Bitfinex's systems and security tokens. By compromising that single device, hackers were able to immediately bypass the BitGo withdrawal limits and drain the wallet. The absence of basic security measures, including server activity logging and a withdrawal whitelist, further contributed to the hack. Bitfinex, however, disputed the report's findings and blamed negligence by other counterparties.
```
Comment (JediFaust) [inline content/attacks/posts/2016-08-02-Bitfinex.md:38]: Remove last sentence

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, simplify and shorten the whole text, since there are unnecessary details. Also try to rewrite some parts with your own words, to pass plagiarism checks.

File: content/attacks/posts/2016-08-02-Bitfinex.md (hunk 1 above)
Comment (EvgDubrovin) [reply content/attacks/posts/2016-08-02-Bitfinex.md]: Done

**Comment by EvgDubrovin** [comment]
> Please, simplify and shorten the whole text, since there are unnecessary details. Also try to rewrite some parts with your own words, to pass plagiarism checks.

1. Rewrote the whole text in the last commit.
2. Removed uninformative parts (mostly from the Timeline).
3. Updated the Headers in accardance with the latest requirements.

**Comment by JediFaust** [comment]
image
Plagiarism issue still remains

File: content/attacks/posts/2016-08-02-Bitfinex.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+Date: 2016-08-02
+Type of attack: "Custodian"
+Target entity: "Bitfinex"
+Type of entity: "Exchange"
```
[the remark this answers is not in the document]
Comment (EvgDubrovin) [reply content/attacks/posts/2016-08-02-Bitfinex.md]: Done

**Comment by EvgDubrovin** [comment]
There is no more plagiarism, according to Grammarly.

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, use [language tool](https://languagetool.org/) and fix grammar issues. Also, merge main into your branch

**Comment by EvgDubrovin** [comment]
> Please, use [language tool](https://languagetool.org/) and fix grammar issues. Also, merge main into your branch

Here is the Language Tool [report](https://drive.google.com/file/d/1180ModmqdiTIQuBnfbKP9AVeUDNDyC4x/view?usp=sharing). 
- I removed the spurious comma. 
- But Language Tool considers "BitGo" and "OCCRP" as spelling mistakes, though these are the names of organizations. 
- I merged main into Bitfinex. There's "Already up to date."

File: content/attacks/posts/2016-08-02-Bitfinex.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2016-08-02
+target-entities: Bitfinex
+entity-types: Exchange
```
Comment (JediFaust) [inline content/attacks/posts/2016-08-02-Bitfinex.md:4]: Add Custodian as an entity type

File: content/attacks/posts/2016-08-02-Bitfinex.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+attack-types: Custodian
```
Comment (JediFaust) [inline content/attacks/posts/2016-08-02-Bitfinex.md:5]: Attack type for the case is `Web Infrastructure Attack'

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, also merge main with the branch

File: content/attacks/posts/2016-08-02-Bitfinex.md (hunk 4 above)
Comment (EvgDubrovin) [reply content/attacks/posts/2016-08-02-Bitfinex.md]: Done

File: content/attacks/posts/2016-08-02-Bitfinex.md (hunk 3 above)
Comment (EvgDubrovin) [reply content/attacks/posts/2016-08-02-Bitfinex.md]: Done

**Comment by EvgDubrovin** [comment]
> Please, also merge main with the branch

Done

File: content/attacks/posts/2016-08-02-Bitfinex.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+date: 2016-08-02
+target-entities: 
+- Bitfinex
+entity-types: 
+- Custodian
```
Comment (JediFaust) [inline content/attacks/posts/2016-08-02-Bitfinex.md:6]: `entity-types` should look like 
> - Custodian
> - Exchange

so, return 'Exchange' back please

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, merge main into the branch

**Review by JediFaust — APPROVED** [review]

**Pull request merged**